rllib/env/wrappers/recsim.py

```diff
@@ -175,7 +175,7 @@ def make_recsim_env(
     """
 
     class _RecSimEnv(gym.Env):
```

Comment:
This should be a gym.Wrapper, not a gym.Env. It probably has no serious impacts, but I've been paranoid about this since we caught that issue with the transformed action space wrapper not seeding properly.

Reply:
Great catch! Will change.

Reply:
Done, saved a bunch of boilerplate :).